style.css

```diff
@@ -0,0 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Noto+Serif+Display:wght@100;200;300;400;500;600;700;800;900&family=Source+Sans+Pro:wght@300;400;600;700;900&display=swap");
+
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+body {
+	min-height: 100vh;
+	font-family: "Courier New", Courier, monospace;
+
+	display: grid;
+	place-content: center;
+	text-align: center;
+
+	background: #ffe4b5;
+}
+
+h1 {
+	font-size: clamp(1rem, 3vw + 1rem, 4rem);
```

Comment:
do you know what does this mean??

Reply:
yes, I know that clamp helpful to make text font responsive for various screen sizes and also it's remove huddle of Writing media query.